spring-cloud-gcp-samples/pom.xml

```diff
@@ -95,6 +95,7 @@
 				<module>spring-cloud-gcp-integration-storage-sample</module>
 				<module>spring-cloud-gcp-trace-sample</module>
 				<module>spring-cloud-gcp-vision-api-sample</module>
+				<module>spring-cloud-gcp-vision-ocr-demo</module>
```

Comment:
Is this "-demo" suffix a distinct concept from "-sample"s? Is it being handled the way you expect w/ native-image-tests.sh?

Reply:
Comparing the code and the READMEs, there doesn't seem to be a distinction between the two. This is the only sample that has the suffix `demo` so it may be a good idea to rename this `sample` to follow convention. cc/ @meltsufin